### DISCUSSION TITLE
Custom Tacotron Waveglow Model or New Model

### DISCUSSION THREAD

**Discussion by WULFFJ** [NVIDIA-NeMo/NeMo#3102] [Q&A]
Goal - to create a custom tts voice or voice in general.  I have everything working properly and installed on my PC.  The tutorials on Colab helped make a ton of sense of the processes for all but creating a new model.  From some reading, I have seen what people call a "warm start", from downloading a model with a little data and adding your own training sets etc.  That would be idea, but I would be happy if I could get a new one as well.  I don't need all details just an idea of what needs done.

**Comment by XuesongYang** [comment]
I would suggest trying to finetune a Fastpitch model on your custom dataset. Please see the tutorial here: https://github.com/NVIDIA/NeMo/blob/main/tutorials/tts/FastPitch_Finetuning.ipynb
